app/assets/javascripts/custom.js

```diff
@@ -0,0 +1,3 @@
+$(function(){
```

Comment:
thoughts on naming this file something more specific? not sure what this lib does, but naming it after what it does would be :+1: :thought_balloon: